lib/controllers/admin_services.ts

```diff
@@ -0,0 +1,217 @@
+/*
```

Comment:
In RESTful APIS, a good convention is to have the controller named after the resource it is responsible for, in case of scoped resources (i.e. `admin/*`) a good convention is to keep the controllers under the scope folder (i.e. `lib/controllers/admin/*`)

Reply:
ok